html/MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include avocado_result_html/resources *
```

Comment:
I don't think we should put this plugin directly as `html`. It's really probable that it'd clash with other `html` module. I think it's fine to put it inside `avocado/plugins` or `avocado-plugins`. Not sure what is a proper place (I know we use `virttest` directly, but this is IMO too much. Consider that this means it'd be directly in `/usr/lib64/python2.6/site-packages`...